src/mw-toast/styles/_mw_toasts.scss

```diff
@@ -114,15 +121,50 @@
             font-weight: normal;
             text-decoration: underline;
           }
+
+          &.success{
+            background: white;
+          }
+
+          &.danger{
+            background: $brand-danger;
+            color: white;
+          }
+
+          &.warning{
+            background: $brand-warning;
+            color: white;
+          }
+
+          &.info{
+            background: $brand-info;
+            color: white;
+          }
+
+          &.primary{
+            background: $brand-primary;
+            color: white;
+          }
         }
 
         .closer {
           position: relative;
-          top: 3px;
+          top: -10px;
+          left: 5px;
           width: 30px;
           text-align: center;
           margin-left: 5px;
           cursor: pointer;
+          opacity: 0.51;
+
+          &.mw-icon {
+            &.success {
+              color: #585858;
```

Comment:
Which color is that? It's better to stick to the available color vars like `$gray`